### PR TITLE
Short list syntax

### PR DESCRIPTION
**Pull request by orklah**
This PR propose to replace the use of list() with the short list syntax []. This is mainly motivated by the fact that the use of PhpStorm refactoring automatically changes those and this pollutes every PR I want to push.

### CODE REVIEW

File: src/Psalm/Internal/Diff/ClassStatementsDiffer.php [hunk 1]
```diff
@@ -109,13 +109,13 @@ function (
                 }
 
                 if ($a instanceof PhpParser\Node\Stmt\ClassMethod && $b instanceof PhpParser\Node\Stmt\ClassMethod) {
-                    if ((string) $a->name !== (string) $b->name) {
+                    if ((string)$a->name !== (string)$b->name) {
```
Comment (weirdan) [inline src/Psalm/Internal/Diff/ClassStatementsDiffer.php:112]: These whitespace changes seem unrelated.

Comment (orklah) [reply src/Psalm/Internal/Diff/ClassStatementsDiffer.php]: You're right, they must have slipped through. I reverted them

**Comment by muglug** [comment]
Thanks!
